lua/bufferline/render.lua

```diff
@@ -207,9 +213,10 @@ local function render(update_names)
 
   local max_scroll = math.max(layout.used_width - layout.buffers_width, 0)
   local scroll = math.min(state.scroll_current, max_scroll)
-  local accumulated_width = 0
   local needed_width = scroll
 
+  accumulated_width = 0
+
```

Comment:
You probably want to revert this?

Reply:
I'm fine with that. The 2 variables are not the same, so the best choice would be to have different names, but short of that redeclaring it clearly says this is a different use. Variable re-use is only ok if it represents the exact same thing.

Reply:
If this function is big enough we're running into issues of clobbering variable names, `render()` probably does too many things…

That would be something for you to decide though, so I'll trust your decision here.